front/app/components/EventCard/EventInformation/index.tsx

```diff
@@ -72,16 +70,22 @@ const EventTimeAndLocationContainer = styled.div`
   ${media.smallerThanMinTablet`
     flex-direction: column;
   `}
-`;
 
-const TimeOrLocation = styled.div<{ first?: boolean }>`
   color: ${(props: any) => props.theme.colorText};
   font-size: ${fontSizes.xs}px;
-  margin-left: ${({ first }) => (first ? 0 : 23)}px;
+`;
+
+const Time = styled.time`
+  ${media.smallerThanMinTablet`
+    margin-bottom: 5px;
+  `}
+`;
+
+const Location = styled.div`
+  margin-left: 23px;
```

Comment:
Same as with margin-bottom, we rather use margin-right on the items on the left. Saying to avoid future clashes/need to refactor.

So in general, margin-bottom/margin-right over top/left, but there are exceptions of course. Maybe it makes sense in your case here, I didn't look at that.